content/asp.net/getting-started/configuring-different-environments.md

```diff
@@ -0,0 +1,153 @@
+# Configuring Different Environments
+by [Eric Fleming](https://ericflemingblog.wordpress.com)
+
+In ASP.NET Core controlling application behavior across multiple environments, such as developement, staging, and production, has been improved through the expanded use of environment variables. Environment variables are used to indicate which environment the application is running in, and can be detected programmatically allowing the application to be configured appropriately.
+
+## Environment Based Settings Files
+The constructor of the Startup class, found below, provides the ability to use more than one appsettings.json file by leveraging these environment variables. Since the `appsettings` configurations are [read in the order they are specified](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration), the general `appsettings` configuration will be loaded first, followed by an environment specific `appsettings` configuration.
+
+```c#
+public Startup(IHostingEnvironment env)
+{
+    var builder = new ConfigurationBuilder()
+        .SetBasePath(env.ContentRootPath)
+        .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+        .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+        .AddEnvironmentVariables();
+    Configuration = builder.Build();
+}
+```
+
+The following loads the environment specific `appsettings.json` configuration;
+
+```c#
+.AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+```
+ 
+where `{env.EnvironmentName}` corresponds to the environment variable `ASPNETCORE_ENVIRONMENT`. When using Visual Studio, the value for the `ASPNETCORE_ENVIRONMENT` variable can be found in your project's debug profiles show below:
+
+![Project properties debug](images/project-properties-debug.png)
+
+and should be set to `Development` by default.
+
+Environment variables will be updated throughout the environments; for example, on Staging the environment variable will likely be set to `Staging`, and for production it will likely be set to `Production`.
```

Comment:
Not sure what this is saying. "will be updated" by whom?
Is this meant to be saying they can be configured for each environment in which the application may be deployed?